extensions/wikia/PortableInfobox/services/PortableInfoboxRenderService.class.php

```diff
@@ -142,14 +142,16 @@ protected function renderGroup( $groupData ) {
 	 */
 	protected function renderImage( $data ) {
 		$helper = $this->getImageHelper();
+
+		$data = $this->filterImageData( $data );
 		$images = [ ];
 
-		for ( $i = 0; $i < count( $data ); $i++ ) {
-			$data[$i]['context'] = null;
-			$data[$i] = $helper->extendImageData( $data[$i], $this->imagesWidth );
+		foreach ( $data as &$dataItem ) {
```

Comment:
Changes on currently iterated array might have unpredicted consequences. Can you do this differently?